packages/components/src/__future__/Select/Select.tsx

```diff
@@ -67,6 +67,10 @@ export type SelectProps<Option extends SelectOption = SelectOption> = {
    * @deprecated: Either define `disabled` in your `Option` (in `items`), or use `disabledKeys`
    */
   disabledValues?: Key[]
+  /**
+   * Creates a portal for the Popover to the matching element id
+   */
+  portalContainerId?: string
```

Comment:
I think instead of confining it to an id, we should expose the same api that we've given the Popover, which accepts `portalContainer` as an element. This way, the consumer can use a ref if they wanted and do not require an id :)

Reply:
This was my suggestion to simplify the API for the most common use case

Reply:
I'm in two minds about this, but having just tested variations of consumer code, I suppose this will be easiest for the consumer since they don't need a `useEffect`. Happy to go this way then :)